### ISSUE TITLE
Issue with as_model

### ISSUE THREAD

**Issue by Jammy2211** [rhayes777/PyAutoFit#444]
The following script runs a test case of a lens modeling pipeline:

https://github.com/Jammy2211/autolens_workspace_test/blob/master/slam/misc/rjlens.py

The stochastic model is made in the following module:

https://github.com/Jammy2211/PyAutoGalaxy/blob/master/autogalaxy/analysis/model_util.py

The specific lines of code are:

```
    model_classes = [MassProfile]
    model = result.instance.as_model(model_classes)

    model.galaxies.lens.take_attributes(source=result.model.galaxies.lens)
```

This has caused no issues previously, but in this test case it loses the `centre` and `elliptical_comps` of the `MassProfile`:

```
        bulge
            intensity                                                                     0.22305566711518723
            effective_radius                                                              0.4624170312787752
            sersic_index                                                                  1.2671926982808182
            mass_to_light_ratio                                                           GaussianPrior, mean = 0.7912556854723145, sigma = 7.912556854723146e-05
```

The issue is the line `model = result.instance.as_model(model_classes)`. The `model` that is output is losing all tuples on the input mass profiles, **when the input tuple is an instance and not a prior**. For example, the tuple on the bulge's `centre` and `elliptical_comps` are lost because they are set via:

```
lens_bulge.centre_0 = 0.0033909063521568973
lens_bulge.centre_1 = -0.009346694405826354
lens_bulge.elliptical_comps.elliptical_comps_0 = 0.0309506766816977
lens_bulge.elliptical_comps.elliptical_comps_1 = -0.0621964387481318
```

In contrast, the `intensity`, `effective_radius` and `sersic_index` are not lost, even though they are set via the same code:

```
lens_bulge.intensity = 0.22305566711518723
lens_bulge.effective_radius = 0.4624170312787752
lens_bulge.sersic_index = 1.2671926982808182
```

This issue is therefore specific to tuples. 

The tuples on the `dark` and `shear` profiles are not lost by this line, becuase they are `model` components as opposed to instances.

In summary, we need `as_model` to work with tuples that are instances!

 following line in

**Comment by rhayes777** [comment]
Digging into this I think centre_0 = syntax shouldn't be used on an instance as it will have no effect.

```python
class MyClass:
    def __init__(self, arg):
        self.arg = arg

instance = MyClass((1, 2))
instance.arg_0 = 3
print(instance.arg)
> (1, 2)

**Comment by rhayes777** [comment]
i.e. 
lens_bulge.centre_0 = 0.0033909063521568973
lens_bulge.centre_1 = -0.009346694405826354

is incorrect syntax
